store/modules/user.module.js

```diff
@@ -0,0 +1,24 @@
+import { usersCollection } from "../../plugins/firebase.js"
+
+export default {
+    namespaced: true,
+    state: {
+        users: []
+    },
+    getters: {
+        usersArray: state => state.users
+    },
+    actions: {
+        async fetchUsers({ commit }) {
+            const data = await usersCollection.get()
+            data.docs.forEach((doc) => {
+                commit('userdetails', doc.data())
+            })
+        },
+    },
+    mutations: {
+        userdetails: (state, data) => {
```

Comment:
naming should be same throughout the app, userdetails should be USER_DETAILS

Reply:
yes sir, corrected